### PR TITLE
README initial version

### PR DESCRIPTION
**Pull request by sargo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    an icon named *Add Build Configuration* should appear near the application name. Click it.
+11. In the *Add Build Configuration* form leave default values in all fields and click the *Build Configuration* button.
+12. Wait several minutes to build.
+13. Open *nRF Connect for Desktop* and open *Programmer* (install it if needed).
+14. Use *Add file* and search for **app_signed.hex** in **build\zephyr** folder of your application.
+15. Select *Nordic Thingy:91* from the *SELECT DEVICE* dropdown.
+16. After clicking *Write* you will see the *MCUboot DFU* popup.
+17. Connect Thingy:91 with a USB cable. Enter boot mode by turning it on while pressing the main button.
+18. Hit the *Write* button in the *MCUboot DFU* popup.
+19. Wait 20-40 seconds unit success message is displayed.
```
Comment (Michal-Jankowski) [inline README.md:40]: Typo in "unit", probably should be "until"

File: README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+20. Turn off and turn on the device to run the application.
```
Comment (Michal-Jankowski) [inline README.md:41]: I wonder, can we change "Turn off and turn on" to "power cycle the device"?

**Comment by Michal-Jankowski** [comment]
I am not the repo maintainer, but from the standpoint of the minimal working example it looks good to me, +1.

**Review by Michal-Jankowski — APPROVED** [review]
I am not the repo maintainer, but from the standpoint of the minimal working example it looks good to me, +1.

File: README.md (hunk 1 above)
Comment (sargo) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (sargo) [reply README.md]: good point

**Pull request merged**